packages/nimble/inst/CppCode/GNUmakefile

```diff
@@ -8,7 +8,12 @@ ifndef R_SHARE_DIR
 endif
 
 # Don't have to succeed in including this if we are calling cleanup in the nimble package directory.
--include $(R_HOME)/etc$(R_ARCH)/Makeconf
+# Following three lines result in much smaller libnimble.a or equivalent by removing -g flags
```

Comment:
Following *two* lines?

Consider converting the human-readable directions into a machine-readable `ifndef NIMBLE_DEBUG_SYMBOLS` or similar.